tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -56,7 +56,7 @@ public async Task GivenUsersExist(Table table)
                         OrganisationFunction = user.OrganisationFunction,
                         SecurityStamp = "TestUser"
                     };
-
+                
```

Comment:
Some unnecessary white space here, not that it will hurt anyone!

Reply:
unnecessary white space hurts everyone